src/scripts/main.js

```diff
@@ -1,3 +1,259 @@
+    };
+
+    newRow.insertAdjacentHTML('beforeend', `
+    <td>${newTd}</td>
+  `);
+
+    if (item.name === 'office') {
+      item.value = newEmployeeForm[2].children[0].value;
+    } else {
+      item.value = '';
+    };
+  });
+
+  tbody.append(newRow);
+
+  pushNotification(10, 10, 'Congratulations',
+    'Employee has been added successfully!', 'success');
+});
+
+const pushNotification = (posTop, posRight, title, description, type) => {
+  const body = document.querySelector('body');
+  const messageInfo = document.createElement('div');
+
+  messageInfo.style.top = `${posTop}px`;
+  messageInfo.style.right = `${posRight}px`;
+  messageInfo.className = `notification ${type}`;
+  messageInfo.dataset.qa = `notification`;
+
+  messageInfo.insertAdjacentHTML('afterbegin', `
+    <h2 class="title">${title}</h2>
+    <p>${description}</p>
+  `);
+
+  body.append(messageInfo);
+
+  setTimeout(() => {
+    messageInfo.remove();
+  }, 2000);
+};
+
+// 5. Editing of table cells by double-clicking
+tbody.addEventListener('dblclick', (eventClick) => {
+  const tdChange = eventClick.target;
+  const tdInnerText = tdChange.innerText;
+  const cellIndex = eventClick.target.cellIndex;
+  const theadInnerText = thead.firstElementChild.children[cellIndex]
+    .innerHTML.toLowerCase();
+
+  if (!tdChange.matches('TD')) {
+    return;
+  };
+
+  const newInput = cellIndex === 2
+    ? document.createElement('select')
+    : document.createElement('input');
+
+  newInput.className = 'cell-input';
+
+  newInput.name = `${theadInnerText}`;
+  newInput.dataset.qa = `${theadInnerText}`;
+  newInput.value = eventClick.target.innerText;
+
+  switch (cellIndex) {
+    case 0:
+    case 1:
+      newInput.type = 'text';
+      newInput.name = `${theadInnerText}`;
+      newInput.dataset.qa = `${theadInnerText}`;
+      break;
+
+    case 2:
+      newInput.name = `${theadInnerText}`;
+      newInput.dataset.qa = `${theadInnerText}`;
+
+      newInput.insertAdjacentHTML('afterbegin', `
+        <option value="Tokyo">Tokyo</option>
+        <option value="Singapore">Singapore</option>
+        <option value="London">London</option>
+        <option value="New York">New York</option>
+        <option value="Edinburgh">Edinburgh</option>
+        <option value="San Francisco">San Francisco</option>
+      `);
+      break;
+
+    case 3:
+      newInput.name = `${theadInnerText}`;
+      newInput.dataset.qa = `${theadInnerText}`;
+      newInput.type = 'number';
+      break;
+
+    case 4:
+      newInput.name = `${theadInnerText}`;
+      newInput.dataset.qa = `${theadInnerText}`;
+      newInput.type = 'number';
+      break;
+  };
+  eventClick.target.innerText = '';
+  tdChange.append(newInput);
+
+  newInput.focus();
+
+  newInput.addEventListener('keydown', (eventKey) => {
+    if (eventKey.key === 'Enter') {
+      newInput.blur();
+    };
+  });
+
+  newInput.addEventListener('blur', () => {
+    switch (true) {
```

Comment:
don't use swithc (true) construction, use if-else instead. You don't need switch construction if you write switch(true)

Reply:
Ok, corrected it